### DISCUSSION TITLE
platform/actors/development/actor-definition/actor-json

### DISCUSSION THREAD

**Discussion by giscus** [apify/apify-docs#638] [Comments]
# platform/actors/development/actor-definition/actor-json

Learn how to write the main actor config in the `.actor/actor.json` file.

https://docs.apify.com/platform/actors/development/actor-definition/actor-json

**Comment by rarous** [comment]
missing `dockerContextDir` property documentation https://docs.apify.com/platform/actors/development/deployment/source-types#actor-monorepos
